lib/floe/workflow/intrinsic_function.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+module Floe
+  class Workflow
+    class IntrinsicFunction
+      class << self
+        def klass(payload)
+          function_module, function =
+            payload.match(/^(?<module>\w+)\.(?<function>\w+)\(.*\)$/)
+              .named_captures.values_at("module", "function")
+
+          case "#{function_module}.#{function}"
+          when "States.StringToJson"
```

Comment:
NOTE trying to avoid a `const_get` here

Reply:
Is there a specific reason why?  We can limit the const_get to just the class we care about if that's the concern.  (e.g. `Floe::Workflow::IntrinsicFunctions.const_get(function_module).const_get(function)`

Reply:
No, I think it's ok, particularly when you are trying to "find" a subclass